gns3/main_window.py

```diff
@@ -1105,9 +1105,28 @@ def startupLoading(self):
                 sys.exit(1)
                 return
 
+        run_as_root_path = LocalConfig.instance().runAsRootPath()
+
         if not sys.platform.startswith("win") and os.geteuid() == 0:
+            # touches file to know that user has run GNS3 as root and to prevent
+            # from running as user
+            if not os.path.exists(run_as_root_path):
+                try:
+                    open(run_as_root_path, 'a').close()
+                except OSError as e:
+                    log.warning("Cannot write `run_as_root` file due to: {}".format(str(e)))
+
             QtWidgets.QMessageBox.warning(self, "Root", "Running GNS3 as root is not recommended and could be dangerous")
 
+        if os.geteuid() != 0 and os.path.exists(run_as_root_path):
+            QtWidgets.QMessageBox.critical(
+                self, "Run as user",
+                "GNS3 has been previously run as root. It is not possible "
+                "to change the user and GNS3 will be shutdown. Please change `{}` directory "
```

Comment:
Should we ask to change permissions on the run_as_root file or ask to delete it?